src/open_inwoner/scss/views/Home.scss

```diff
@@ -0,0 +1,11 @@
+/// Cards on Home Page and Theme page
+
+.home {
+  .card-container {
+    grid-template-columns: repeat(auto-fit, 228px);
```

Comment:
Maybe we could add a specific rule for the mobile version as it seems a bit weird to my eyes, but you know better.

![image](https://user-images.githubusercontent.com/63058071/227239749-959eeea1-21a1-41cc-8a6c-bed0d2d7dd78.png)

Reply:
@vaszig For mobile: horizontal scrolling will be added later in a different issue so it will look better, but the cards will need to remain this same size.

Reply:
What do you think @alextreme ?

Reply:
This is fine for now, the scrolling is something Jiro will continue on but these changes are a necessary first step for that.